### PR TITLE
[ACA-4227] [APS] Create a script to check Process Services Management plugin status before running e2e tests

### PR DESCRIPTION
**Pull request by sivakumar414ram**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [x] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/ACA-4227

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/cli/scripts/plugins/aae-plugin.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    config: PluginConfiguration;
+
+    constructor(
+        private plugInInfo: PluginInterface,
+        private alfrescoJsApi: any
+    ) {
+        this.config = new PluginConfiguration(
+            this.plugInInfo,
+            this.alfrescoJsApi,
+            true
```
Comment (mauriziovitale) [inline lib/cli/scripts/plugins/aae-plugin.ts:15]: a method should never have a boolean. 
if you have a boolean means that you have 2 behaviour... so you can create 2 classes.

File: lib/cli/scripts/plugins/aps-plugin.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { PluginInterface } from './plugin-model';
+import { logger } from '../logger';
+import { PluginConfiguration } from './plugin-config';
+
+export class ProcessServicePlugin {
+    config: PluginConfiguration;
+
+    constructor(private plugInInfo: PluginInterface, private alfrescoJsApi: any) {
+        this.config = new PluginConfiguration(this.plugInInfo, this.alfrescoJsApi, false);
```
Comment (mauriziovitale) [inline lib/cli/scripts/plugins/aps-plugin.ts:9]: a method should never have a boolean. 
if you have a boolean means that you have 2 behaviour... so you can create 2 classes.

File: lib/cli/scripts/plugins/aps-plugin.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { PluginInterface } from './plugin-model';
+import { logger } from '../logger';
+import { PluginConfiguration } from './plugin-config';
+
+export class ProcessServicePlugin {
```
Comment (mauriziovitale) [inline lib/cli/scripts/plugins/aps-plugin.ts:5]: can we rename the file into process-service-check-plugin?

File: lib/cli/scripts/plugins/aae-plugin.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import { PluginInterface } from './plugin-model';
+import { logger } from '../logger';
+import { PluginConfiguration } from './plugin-config';
+
+export class ProcessAutomationPlugin {
```
Comment (mauriziovitale) [inline lib/cli/scripts/plugins/aae-plugin.ts:5]: can we rename the file into process-automation-check-plugin?

File: lib/cli/scripts/plugins/aae-plugin.ts (hunk 1 above)
Comment (sivakumar414ram) [reply lib/cli/scripts/plugins/aae-plugin.ts]: Updated

File: lib/cli/scripts/plugins/aps-plugin.ts (hunk 2 above)
Comment (sivakumar414ram) [reply lib/cli/scripts/plugins/aps-plugin.ts]: Yup, Updated

File: lib/cli/scripts/plugins/aps-plugin.ts (hunk 3 above)
Comment (sivakumar414ram) [reply lib/cli/scripts/plugins/aps-plugin.ts]: Updated

File: lib/cli/scripts/plugins/aae-plugin.ts (hunk 4 above)
Comment (sivakumar414ram) [reply lib/cli/scripts/plugins/aae-plugin.ts]: Updated

**Review by mauriziovitale — APPROVED** [review]

**Pull request merged**